packages/config/src/android/Updates.ts

```diff
@@ -9,6 +22,10 @@ export function getUpdateUrl(config: ExpoConfig, username: string | null) {
   return `https://exp.host/@${user}/${config.slug}`;
 }
 
+export function getRuntimeVersion(config: ExpoConfig) {
```

Comment:
```suggestion
export function getRuntimeVersion(config: ExpoConfig): string | null {
```

Reply:
I'll leave this for now for consistency since none of the other functions have an annotation.